db/data/20190829135327_backfill_first_middle_surnames.rb

```diff
@@ -0,0 +1,26 @@
+class BackfillFirstMiddleSurnames < ActiveRecord::Migration[5.2]
+  def up
+    claims_with_full_names.each do |claim|
+      parser = Claim::VerifyResponseParametersParser.new(claim.verify_response)
```

Comment:
Nice, having the parser broken out is super useful here 😃

Reply:
Yeah it made things super simple 👍 thanks @tekin